tensorflow/core/kernels/reverse_op.cc

```diff
@@ -149,9 +145,13 @@ TF_CALL_double(DECLARE_GPU_SPEC);
                           ReverseOp<GPUDevice, T>)
 TF_CALL_uint8(REGISTER_GPU_KERNEL);
 TF_CALL_int8(REGISTER_GPU_KERNEL);
+//TF_CALL_int32(REGISTER_GPU_KERNEL);
+//TF_CALL_bool(REGISTER_GPU_KERNEL);
```

Comment:
Why are these commented out?  If there are broken reasons why they can't be added, maybe leave them commented out but add a TODO comment saying why.